src/phases/query/TimeSeriesLastpoint.yml

```diff
@@ -92,3 +83,17 @@ DropIndex:
     OperationCommand:
       dropIndexes: {^Parameter: {Name: "Collection", Default: "Collection0"}}
       index: {^Parameter: {Name: "IndexName", Default: ""}}
+
+QuiesceActor:
+  Name: {^Parameter: {Name: "Name", Default: "FindOne"}}
+  Type: QuiesceActor
+  Threads: 1
+  Database: {^Parameter: {Name: "Database", Default: admin}}
+  Phases:
+    OnlyActiveInPhases:
+      Active: {^Parameter: {Name: "Active", Default: [0, 2, 4, 6, 8, 10]}}
```

Comment:
Can we make the default ```[2, 5, 8, 11, 14, 17, 18, 21]``` and then remove Active from the call site?

Reply:
Definitely better than having random defaults, fixed